src/pages/extensions.md

```diff
@@ -59,24 +59,24 @@ OpenRefine can connect to several reconciliation services. You can find informat
 
 ## Client libraries
 
-You can automate some OpenRefine operations using one of the existing libraries. Those libraries are using the [OpenRefine API](https://github.com/OpenRefine/OpenRefine/wiki/OpenRefine-API).
+You can automate some OpenRefine operations using one of the existing libraries. Those libraries are using the [OpenRefine API](/docs/technical-reference/openrefine-api).
```

Comment:
"You can automate" is an invitation to users to shoot themselves in the foot. We need a strong warning here about the risks involved.

```suggestion
There are a number of unsupported client libraries which use the internal [OpenRefine client/server protocol](/docs/technical-reference/openrefine-api). Because these use an unsupported protocol which has no tests, these libraries should only be used by technically sophisticated developers who understand the risks and limitations. These libraries are not provided or supported by the OpenRefine team.
```

(I know that this is existing wording and not something changed by this PR)